src/main/java/co/cask/gcp/spanner/source/SpannerSource.java

```diff
@@ -132,6 +137,16 @@ public void prepareRun(BatchSourceContext batchSourceContext) throws Exception {
     // set input format and pass configuration
     batchSourceContext.setInput(Input.of(config.referenceName,
                                          new SourceInputFormatProvider(SpannerInputFormat.class, configuration)));
+    if (config.getSchema() != null) {
+      if (config.getSchema().getFields() != null) {
+        FieldOperation operation =
+          new FieldReadOperation("Read", "Read from Spanner table",
```

Comment:
Sometimes the description has a trailing dot. Other times, it doesn't.
Let's make them consistent.

Reply:
Lets do it in an independent PR against 0.11 so that can be pulled to develop directly.